doc/documents/Examples_Tutorials/Examples_Tutorials.rst

```diff
@@ -0,0 +1,623 @@
+Examples and Tutorials
+======================
+
+Model Deployment Tutorial for Caffe and CIFAR-10
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this chapter, we guide you through the manual model deployment process taking CIFAR-10 application as an example. The example considered is based on Caffe standard tutorial. We are not limited to Caffe and similar approach might be applied to other frameworks. Helper scripts considered here are also available in the repo.
+
+Our assumption is that you are familiar with: 
+
+ - **Neural networks basics** – convolutions, tensors, ReLU, and so on
+
+ - **Python** – you can read and understand python code
+
+ - **NumPy** – de-facto standard numerical library for python
+
+ - **С programming language** – MLI API is C level API
+
+ - **Caffe framework basics**
+ 
+The development process of MLI-based embedded application is depicted with diagram:
+
+.. image::  ../images/1_depl_process.png
+   :align: center
+   :alt: MLI-Based Application Development Process
+
+..
+
+1. Model definition and training in some appropriate framework. Ensure that you consider all limitations of the target platform here including memory restriction, MHz budget, and quantization effect in some cases.
```

Comment:
It's better to remove ".. , and quantization effect in some cases" as it's more a recomendation, while sentence in this form  seems like requirement.